### ISSUE TITLE
RED shows a wrong usage exception by a correct used varargs Keyword from a custom java library

### ISSUE THREAD

**Issue by JKotik** [nokia/RED#256]
Hello,

RED shows a "Keyword '_Key Includes only_' expects 0 non-keyword arguments but **x** are provided" exception by providing the keyword _Key Includes only_ with **n** arguments. 
_Includes only_ is defined as
`public void  keyIncludesOnly (String... args)
{
...
}`
The definition is correct and the test is executed as expected (after comforming of "Error in Workspce" warning)

**Comment by JKotik** [comment]
`
*** Setting ***
Metadata                   Version        0
Library                       net.xxx.xxx.xxx.testmodules.KeyLibrary 
Documentation         someText
Default Tags              aTag



*** Test Cases ***
Key.1: Meaningful name of the first test
    [Documentation]                  someText       
    [TAGS]                                  Tag1    Tag2    Tag3     
              
    One method from lib           ${CURDIR}/temp/xxx.xxx       
    Key Includes only                  arg1    arg2   `

**Comment by michalanglart** [comment]
Hi,

sorry for long delay. Can you also provide also library specification file for this library? It is placed inside `libspecs` directory inside your project.

**Comment by michalanglart** [comment]
This is a bug.

I was able to reproduce it. Don't know however yet if that's a problem in RED or in Robot.

Reproduction:
- create java library `com.example.Library` with following method:
```
public void doSomething(final String... args) { ... }
```
- build `.jar` file with library
- run:
```
jython -J-cp ".;path/to/jar/file/com.example.Library.jar" -m robot.libdoc com.example.Library show
```
- the output would be:
```
...
Do Something
____________
Arguments: [**varargs]
```

So the arguments descriptor shows that `Do Something` keyword accepts `kwargs` (two asterisks) instead of `varargs` (should be single asterisk, name of argument doesn't matter). This behavior results in RED wrongly validating provided arguments.

The only question is if it's possible for Robot to properly show that `Do Something` accepts `[*varargs]`. The User Guide suggests that `libdoc` should be run over java sources in case of static libraries instead of compiled binaries (http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#general-usage paragraph `Java libraries with path`).

**Comment by michalanglart** [comment]
This problem was fixed on RobotFramework side and will be released with RF 3.1 - probably quite soon. 

However there still remains a little bit related issue: #218  where RED does not handle java sources in order to properly generate library documentations.

I'm closing this as there is nothing more we can do on RED side - for now use the workaround and just update to RF 3.1 as soon as it will be released
